### PR TITLE
Mobile project

### PR DESCRIPTION
**Pull request by Daron976**
## Mobile Project 

- In this project:
   - Added the work and about sections
   - styled the work and about sections

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -110,8 +112,117 @@ a {
...
+}
+
+.work_one,
+.work_two,
+.work_three,
+.work_four {
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+}
```
Comment (levy002) [inline styles.css]: - [ ] Kindly make sure to stick to the provided Figma design by using the correct fonts and please make sure to add margins in order to meet the design 😉. You can import the font from the [google font](https://fonts.google.com) site.
- [ ] Make sure to place your background images in the right position as shown in the Figma design.

   - ### Your design
  project


   - ### Figma design
  real-work

File: index.html [hunk 2]
```diff
@@ -72,8 +70,106 @@ <h3 class="ability_header">Skills</h3>
...
+                <div class="btn_links">
+                    <button type="button" class="btn">See Project</button>
+                    <ul class="social_media_links">
+                        <li class="left_border"><i class="fa-brands fa-github"></i></li>
+                        <li><i class="fa-brands fa-linkedin"></i></li>
+                        <li><i class="fa-brands fa-twitter"></i></li>
+                        <li><i class="fa-brands fa-angellist"></i></li>
+                    </ul>
+                </div>
+            </section>
```
Comment (levy002) [inline index.html:173]: - Kindly add the `email` as it is visible to the provided Figma design. 😉 

  - ### Figma design
  lololol


    - ### Your design
  rals

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @Daron976, 

### I'm Levy, your PR reviewer.

Good job so far! 😉 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linters are passing ✔️ 
- Good commit history ✔️ 
- Descriptive README ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @levy002 me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -72,8 +70,105 @@ <h3 class="ability_header">Skills</h3>
...
+                <div class="btn_links">
+                    <button type="button" class="btn">See Project</button>
+                    <ul class="social_media_links">
+                        <li class="left_border"><i class="fa-brands fa-github"></i></li>
+                        <li><i class="fa-brands fa-linkedin"></i></li>
+                        <li><i class="fa-brands fa-twitter"></i></li>
+                        <li><i class="fa-brands fa-angellist"></i></li>
+                    </ul>
+                    <p>baronjones976@gmail.com</p>
+                </div>
```
Comment (MaganaAsiati) [inline index.html:171]: - [ ] [ OPTIONAL ] kindly remove this contact section is not part of the requirement  to add it, you will implement it in the next project, here you will implement only card section :grinning: :grinning:

File: index.html (hunk 2 above)
Comment (MaganaAsiati) [inline index.html:172]: - kindly buy trying to fix this email requires by the previous review you add another bag

> Kindly add the email as it is visible to the provided Figma design.


- kindly make sure you fix this, your icons are not showing well, they combine with the title of  your card 
image

File: index.html [hunk 4]
```diff
@@ -72,8 +70,105 @@ <h3 class="ability_header">Skills</h3>
...
+                    </p>
+                    <ul class="languages">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button type="button" class="btn">See Project</button>
+                </div>
+            </div>
```
Comment (MaganaAsiati) [inline index.html:92]: Kindly make sure your card looks like the Figma design. Add margins (top, bottom, left, right) defined in the design as defined in the design. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]
## your design
image



## Figma design

image

File: styles.css [hunk 5]
```diff
@@ -110,8 +114,136 @@ a {
...
+.work ul {
+  margin-bottom: 1.5rem;
+}
+
+.btn {
+  font-size: 0.85rem;
+  color: white;
+  background-color: #9390ff;
+  border: none;
+  border-radius: 5px;
```
Comment (MaganaAsiati) [inline styles.css]: - [ ] kindly remove the border raduis,  to make your button match with the button design in  the figma .
## your button 
image
## button in figma design
image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# hi @Daron976 
# STATUS: Required Changes ♻️♻️♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional
✅ descriptive pull request 

## Required Changes ♻️
- [ ] Great job on styling your page, it looks awesome 100 However, we are required to follow the Figma design.  You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]👍🏻

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 3 above)
Comment (Daron976) [reply index.html:171]: kindly note that the email section you are asking is at the end of the contact page you want me to remove

File: styles.css (hunk 1 above)
Comment (mgmediaweb) [reply styles.css]: Kindly you need to work with what the previous reviewer commented
_"Make sure to place your background images in the right position as shown in the Figma design."_

File: styles.css (hunk 5 above)
Comment (mgmediaweb) [reply styles.css]: Good job following the advice of the previous reviewer.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi  @Daron976, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

*** Kindly you need to work with what the previous reviewer commented
"Make sure to place your background images in the right position as shown in the Figma design."

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Daron976, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

### Figma Design
image

### Your Proposal
Kindly, the backlights are not displayed, please work with that
image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — APPROVED** [review]
Hi @Daron976, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved6

## Optional suggestions

[OPTIONAL] Kindly, the background you put is static, I recommend you use multiple backgrounds that adapt to all devices. https://www.w3schools.com/css/css3_backgrounds.asp

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**